specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.typescript.md

```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-containerservicefleet"
```

Comment:
New readme.typescript for fleet, modified `package-name` and `output-folder` to append fleet.